config/sync/node.type.campaign.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 37edecd9-3cc1-4ab7-81b9-7a34d0c2c913
+langcode: da
+status: true
+dependencies: {  }
+name: Kampagne
+type: campaign
+description: 'Kampagne der kan vises på søgeresultater med mathende facetter og termer'
```

Comment:
Please ensure that there configurations are in English by default and translateable to Danish.

Reply:
I thought that when langcode is "da" the name and description should be represented in the same language.
Like it is the case of the promoted field, which has all been generated by the system:
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/143/files/6ef736f396a1ba614baee15f74ca3e6201997f9e#diff-44ed5b104914630c7c07b5eea2a98403cc5024925256512d337c2f3eea57b081

Reply:
Language export problems should have been solved by this commit:
d0302ef6ce9713e69c322655e82ed0e302e964ec